app/controllers/activitypub/outboxes_controller.rb

```diff
@@ -4,7 +4,7 @@ class ActivityPub::OutboxesController < Api::BaseController
   before_action :set_account
 
   def show
-    @statuses = @account.statuses.permitted_for(@account, current_account).paginate_by_max_id(20, params[:max_id], params[:since_id])
+    @statuses = @account.statuses.not_local_only.permitted_for(@account, current_account).paginate_by_max_id(20, params[:max_id], params[:since_id])
```

Comment:
It might be a bit cleaner to do the `not_local_only` inside `Status#permitted_for`.

I also wonder whether we do want to be hiding these from logged-in local users, as unconditionally doing `not_local_only` here does? The web UI never fetches the outbox I think, so it won't cause problems necessarily, but I was assuming we'd leave it returning them for authenticated local users.

Reply:
We don't really want to be hiding them from logged in local users no. Can you suggest or make the fix and push it to the branch please?

Reply:
I didn't see a clean way to fit it into `Status#permitted_for`. Feel free to push to the branch.